pkg/backends/calico/routes.go

```diff
@@ -98,20 +97,12 @@ func (rg *routeGenerator) Start(routeString string) {
 	// parse routeString
 	for _, route := range strings.Split(routeString, ",") {
```

Comment:
I think we can get rid of this loop altogether - routeString isn't a list anymore.